mysql-test/suite/rpl/r/rpl_get_master_version_and_clock.result

```diff
@@ -11,6 +11,8 @@ call mtr.add_suppression("Get master clock failed with error");
 include/stop_slave.inc
 SET @@global.debug= "+d,dbug.before_get_UNIX_TIMESTAMP";
 START SLAVE;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
The same as above, but the warning comes from extra/rpl_tests/rpl_get_master_version_and_clock.test